morphic.js

```diff
@@ -4496,6 +4496,56 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
+    
+    // Add hidden text box for copying and pasting
+    this.hiddenText = document.createElement('textarea');
+    this.target.hiddenText = this.hiddenText;
```

Comment:
why do we need to cache the hidden text in the target? Won't that reference stick around even and be orphaned if the hidden text gets deleted?